.github/workflows/test_rust.yml

```diff
@@ -6,6 +6,7 @@ on:
   pull_request:
     branches: [master]
 
+
```

Comment:
Please remove newly-added excess newlines (this one and after `Upload playerglobal` step).